apps/etterlatte-grunnlag/src/main/kotlin/grunnlag/BehandlingGrunnlagRoutes.kt

```diff
@@ -84,10 +84,12 @@ fun Route.behandlingGrunnlagRoute(
         }
 
         post("oppdater-grunnlag") {
-            withBehandlingId(behandlingKlient) { behandlingId ->
-                val request = call.receive<OppdaterGrunnlagRequest>()
-                grunnlagService.oppdaterGrunnlag(behandlingId, request.sakId, request.sakType)
-                call.respond(HttpStatusCode.OK)
+            kunSystembruker {
+                withBehandlingId(behandlingKlient) { behandlingId ->
+                    val request = call.receive<OppdaterGrunnlagRequest>()
+                    grunnlagService.oppdaterGrunnlag(behandlingId, request.sakId, request.sakType)
+                    call.respond(HttpStatusCode.OK)
+                }
```

Comment:
Mulig det er et eller annet jeg overser, men dette vil da feile? Så vidt jeg vet er det saksbehandler som trigger oppdatering av grunnlag, _ikke systembruker_...?

Reply:
Har endret på det, for å få med statusendring 😛 SB poster nå til `/behandling/${args.behandlingId}/oppdater-grunnlag` og så kaller behandling på grunnlag med systembruker-token før statusen settes til Opprettet på behandlingen.